packages/fluentui/react-northstar/src/components/SplitButton/SplitButtonToggle.tsx

```diff
@@ -0,0 +1,174 @@
+import { Accessibility, buttonBehavior } from '@fluentui/accessibility';
+import * as customPropTypes from '@fluentui/react-proptypes';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import {
+  childrenExist,
+  createShorthandFactory,
+  UIComponentProps,
+  ChildrenComponentProps,
+  commonPropTypes,
+  rtlTextContainer,
+  ContentComponentProps,
+} from '../../utils';
+
+import {
+  ComponentEventHandler,
+  WithAsProp,
+  withSafeTypeForAs,
+  FluentComponentStaticProps,
+  ProviderContextPrepared,
+} from '../../types';
+import { getElementType, useAccessibility, useStyles, useTelemetry, useUnhandledProps } from '@fluentui/react-bindings';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+
+export interface SplitButtonToggleProps extends UIComponentProps, ContentComponentProps, ChildrenComponentProps {
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility;
```

Comment:
```suggestion
  accessibility?: Accessibility<never>;
```

Reply:
Actually I am using here `buttonBehavior`, so it probably should be `ButtonBehaviorProps`. Should Is pecify this or create new behavior that will invoke the `buttonBehavior`?

Reply:
`ButtonBehaviorProps` is fine